deployments/liqo/README.md

```diff
@@ -18,8 +18,10 @@
 | auth.pod.extraArgs | list | `[]` | Extra arguments for the auth pod. |
 | auth.pod.labels | object | `{}` | Labels for the auth pod. |
 | auth.pod.resources | object | `{"limits":{},"requests":{}}` | Resource requests and limits (https://kubernetes.io/docs/user-guide/compute-resources/) for the auth pod. |
+| auth.service.allocateLoadBalancerNodePorts | string | `""` | If service type is LoadBalancer and want to expose the gateway on NodePorts set it to true |
```

Comment:
```suggestion
| auth.service.allocateLoadBalancerNodePorts | string | `""` | Set to 'true' if the service type is LoadBalancer and you want to expose the gateway on NodePorts. |
```
Additional question. If I understand correctly, this parameter could be better named as 'auth.service.allocateGatewayNodePorts', is it correct?
Otherwise it seems to me I did not understand what this flag is intended to do.

Reply:
You are right. Need to change from auth to Gateway. Will update the PR.